src/_gettsim_tests/test_full_taxes_and_transfers.py

```diff
@@ -49,23 +48,11 @@ def test_full_taxes_and_transfers(
     if test_data.date.year <= 2008:
         out.remove("abgelt_st_y_tu")
 
-    # TODO(@hmgaudecker): Remove again once unterhaltsvors_m is implemented
```

Comment:
I moved this to-do to #479 and #585. Would be great to ping the owner of a to-do before deleting it -- better safe than sorry!